interface/src/components/App/PermanentDrawer.tsx

```diff
@@ -323,10 +323,6 @@ export function PermanentDrawer(props: any): any {
                                     </List>
                                 </div>
                             }
-                            <div className="plus-full-circle">
```

Comment:
Please put back this code and and you can try to implement the add new contact functionality.